Clinical-Genomics/scout
# Improved genes info for large  SVs and cancer SVs

Current situation is that whenever a large SV (or cancer SV) has more than 30 genes, only the first 30 are shown. This is problematic in many cases for the users (fix #2887).

This PR adds the possibility to toggle the display of all genes for large SVs, both on SV and cancer SV variantS page;

<img width="600" alt="image" src="https://user-images.githubusercontent.com/28093618/184130724-0f6bffd9-d090-4172-af7f-6aca83877857.png">

And on SV variant page (templated shared by RD and cancer Sv vars:

<img width="306" alt="image" src="https://user-images.githubusercontent.com/28093618/184131193-7158f937-e34d-4b75-a781-cb10c1caf136.png">

<img width="362" alt="image" src="https://user-images.githubusercontent.com/28093618/184131613-b0cf7eb0-5f62-4c75-9d69-7cef354b7ef0.png">


(both on SV variantS page and on SV variant page)

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>


**How to test**:
1. Install on cg-vm1 and go check SVs variantS pages for both RD cases and recent cancer cases (variants with > 5 genes)
2. Check SV variant page for SV variants with > 30 genes

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
